src/ZEO/ClientStorage.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, addr, storage='1', cache_size=20 * MB,
                  wait=True,
                  drop_cache_rather_verify=True,
                  username=None, password=None, realm=None,
```

Comment:
These are left over from the old ZEO 4 authentication. I kept them for backward compatibility, but it's probably better to remove them. I'll do that in a separate PR.